particles/Tutorial/Kotlin/Demo/src/pt1/TTTGame.arcs

```diff
@@ -3,10 +3,7 @@
 
 
 schema GameState
-  gameOver: Boolean
   board: Text
-  currentPlayer: Number
-  winnerAvatar: Text
```

Comment:
I thought I'd taken everything out in the last PR, but in writing the tutorial, I realized it was easier to also not have these fields when starting to create the game.